man/scrot.txt

```diff
@@ -3,8 +3,8 @@ NAME
 
 SYNOPSIS
   scrot [-bcfhimopuvz] [-a X,Y,W,H] [-C NAME] [-D DISPLAY] [-d SEC] [-e CMD]
-        [-F FILE] [-k OPT] [-l STYLE] [-M NUM] [-n OPTS] [-q NUM] [-S CMD] [-s OPTS]
-        [-t % | WxH] [FILE]
+        [-k OPT] [-l STYLE] [-M NUM] [-n OPTS] [-q NUM] [-S CMD] [-s OPTS]
+        [-t % | WxH] [FILE | -F FILE]
```

Comment:
This looks really weird. I don't think I've ever seen non-option arg and option arg be mixed like this before.

Reply:
How about imitating st's manual here then? https://manpages.debian.org/bullseye/stterm/st.1.en.html

It has an `-e` option which specifies a command for the terminal to run on startup. It can also be specified as a non-option argument, so the manual's synopsis is `[[-e] command [arguments...]]`.

Reply:
> How about imitating st's manual here then?

ST manual is a bit different than this though. `-e` in ST isn't mutually exclusive with non-option argument.

But what exactly do you have in mind? I think I can give a better answer if I see a more concrete example.

Reply:
I guess `[[-F] FILE]` would be ok.